scripts/perftune.py

```diff
@@ -1351,7 +1348,7 @@ def dump_config(prog_args):
     if prog_args.mode:
         prog_options['mode'] = prog_args.mode
 
-    if prog_args.nic:
```

Comment:
There is an `nic` option, but the value is saved to `args.nics`.
I'm ok with this change.